redhat-director-scripts/puppet/trilio/manifests/contego.pp

```diff
@@ -16,7 +16,7 @@
     $contego_group                        = 'nova'
     $contego_conf_file                    = "/etc/tvault-contego/tvault-contego.conf"
     $contego_groups                       = ['kvm','qemu','disk']
-    $vault_data_dir                       = "/var/triliovault-mounts"
+    $vault_data_dir                       = "/var/lib/nova/triliovault-mounts"
```

Comment:
So on director, it is always going to be /var/lib/nova as mount point, right?

Reply:
For now, I create separate branch on upstream also. But if want to merge it in master then we need to make backup target mountpoint directory configurable from configurator.
That's how it will be available for all customers.